### PR TITLE
Added parser support for EK80 MRU1

### PR DESCRIPTION
**Pull request by praneethratna**
Addresses #1239 and now EK80 files of type MRU version 1 can be parsed.

CC @leewujung

### CODE REVIEW

File: echopype/convert/utils/ek_raw_parsers.py [hunk 1]
```diff
@@ -522,7 +522,17 @@ def __init__(self):
                 ("roll", "f"),
                 ("pitch", "f"),
                 ("heading", "f"),
-            ]
+            ],
+            1: [
```
Comment (leewujung) [inline echopype/convert/utils/ek_raw_parsers.py:526]: Could you add in the docstring regarding MRU1 type in addition to MRU0?

**Review by leewujung — COMMENTED** [review]
@praneethratna : Thanks for putting this in! I just have a minor comment about adding MRU1 in the docstring. I verified that the rest of `open_raw` runs through fine with the pitch, roll, heave data stored in the Platform group as pitch, roll, and _vertical_offset_.

But the parsed `heading` variable is not stored. Let me take a look at the convention to see if there's a variable we can store it, and if not, we can put it in the Platform group just with `heading` as the name. I will get back to you on this!

It'll be nice to have a test file in the CI for this.

@jmjech: Do you have a file that is smaller (say a few MB) that we can use as a test file? It adds to the time for tests to run through if the test files are large, so we usually try to include only small files.

Thanks both!

**Comment by jmjech** [comment]
I e-mailed a 2 MB file to Wu-Jung. Let me know if it works for you. I have other "smaller" files.
For some reason it is not allowing me to add the data files here.

**Comment by leewujung** [comment]
@jmjech: Thank you! The file works and does contain MRU1 datagrams.

@praneethratna: I have added this file (`20231016_Cal_-D20231016-T220322.raw`) under `test_data/ek80`. Could you please add a test for parsing it? In looking into this file I also checked for `np.all(echodata["Platform"]["pitch"].data == np.array(parser.mru["pitch"]))` and the same thing for `heave` (compare with `vertical_offset` in Platform) and `roll` (compare with `roll` in Platform). Would be good if we have these in the tests, by using the parser object https://github.com/OSOceanAcoustics/echopype/blob/03e3b52bb522222f5cdf887b6f519e47d4aae6bf/echopype/convert/api.py#L423

**Comment by leewujung** [comment]
Also, on storing the `heading` variable. It is indeed in the ver.1 convention as below:

Screenshot 2023-12-14 at 11 43 23 AM

@praneethratna: Could you please add this to the set group so that it gets stored together with the other variables? As you have seen, they are on the same timebase since they are all from the MRU* datagrams. Thanks!

File: echopype/convert/utils/ek_raw_parsers.py (hunk 1 above)
Comment (praneethratna) [reply echopype/convert/utils/ek_raw_parsers.py:536]: Updated!

**Comment by praneethratna** [comment]
@leewujung I have added docstring for new `MRU1` datagram and also added a new test case using the file given by @jmjech. Also, now the `heading` variable will be stored in `Platform` group with the attributes which you've mentioned. Thanks!

**Review by leewujung — APPROVED** [review]
@praneethratna : Thanks for adding `heading` and test. I'll merge this now. :)

**Pull request merged**